src/constants.py

```diff
@@ -32,7 +32,7 @@
 TLS_SSL_CERT_FILE = "custom-server-cert.pem"
 MYSQL_EXPORTER_PORT = "9104"
 CHARMED_MYSQL_SNAP_NAME = "charmed-mysql"
-CHARMED_MYSQL_SNAP_REVISION = 51
+CHARMED_MYSQL_SNAP_REVISION = 65
```

Comment:
TODO: update revision once new revision of snap is published to 8.0/edge once https://github.com/canonical/charmed-mysql-snap/pull/35 is merged

Reply:
updated to latest 8.0/edge (revision 66) in 9f7d5c199f5e6824c0f44babf3c7917c58930592